frontends/web/src/app.jsx

```diff
@@ -95,22 +95,16 @@ export class App extends Component {
     }
 
     onDevicesRegisteredChanged = () => {
-        apiGet('devices/registered').then(deviceIDs => {
-            this.setState({ deviceIDs });
+        apiGet('devices/registered').then(devices => {
+            const deviceIDs = Object.keys(devices);
+            this.setState({ devices, deviceIDs });
         });
     }
 
     onAccountsStatusChanged = () => {
         apiGet('accounts-status').then(status => {
             const accountsInitialized = status === 'initialized';
-            this.setState({
-                accountsInitialized
-            });
-            if (!accountsInitialized) {
-                console.log('app.jsx route /'); // eslint-disable-line no-console
-                route('/', true);
```

Comment:
No, that is because they are activated in device.jsx after unlock. Similar should be done in the software based one, or move this to a more generic place.

Btw. this particular lines I restored later, but adjusted to only apply when you are in /accounts/ (otherwise it would override othe redirects, e.g. to the inserted device, in a race).